frame/babe/src/default_weights.rs

```diff
@@ -0,0 +1,47 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Default weights for the Babe Pallet
+//! This file was not auto-generated.
+
+use frame_support::weights::{
+	Weight, constants::{WEIGHT_PER_MICROS, WEIGHT_PER_NANOS, RocksDbWeight as DbWeight},
+};
+
+impl crate::WeightInfo for () {
+	fn report_equivocation(validator_count: u32) -> Weight {
+		// we take the validator set count from the membership proof to
+		// calculate the weight but we set a floor of 100 validators.
+		let validator_count = validator_count.max(100) as u64;
+
+		// worst case we are considering is that the given offender
```

Comment:
@andresilva you should keep an eye on this: https://github.com/paritytech/substrate/issues/6641

TLDR; It is really nonsensical that we store all backers of a validators and slash all of them but reward only a portion of them (top 64). I am considering making this always top 64, so here you can always ensure that `MAX_NOMINATORS  = 64;`

Reply:
Good! That way we can drop this assumption and have a strict upper bound on the number of nominators a slash will affect.